packages/components/ui/src/gauges/screen/ScreenGauge.svelte

```diff
@@ -0,0 +1,140 @@
+<script context='module'>
+	import {writable} from 'svelte/store';
+
+	export const screen = writable();
+</script>
+
+<script>
+	import * as _ from 'lamb';
+	import {getTruthyValuesKeys, joinWithBlank} from '@svizzle/utils';
+
+	const getClasses = _.pipe([getTruthyValuesKeys, joinWithBlank]);
+	const sampleText = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
```

Comment:
Maybe export `sampleText`  for allowing users to provide their own text if (i.e. using different alphabets)

Reply:
nice idea, ok